### ISSUE TITLE
How can I delete an unwanted dependency from the metadata?

### ISSUE THREAD

**Issue by chrisr3** [jjohannes/gradle-module-metadata-maven-plugin#13]
The `maven-shade-plugin` plugin has these configuration options:
```xml
<configuration>
    <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
    <createDependencyReducedPom>true</createDependencyReducedPom>
</configuration>
```
which modify dependencies in the published POM for those dependencies which have been shaded into the final jar. However, the `gradle-module-metadata-maven-plugin` doesn't seem to realise this.

The plugin is already listed _after_ `maven-shade-plugin` in the POM, but this makes no difference to the published `module`.

**Comment by jjohannes** [comment]
Thanks for raising the issue. It's not possible to do this yet (remove a dependency). But looks like it's needed when combining this plugin with others like the _shade_ plugin. I will add that feature and create a new release.

**Comment by jjohannes** [comment]
Resolved in f8f8e5a26ed17bb629f1fcab669ddc70a2b6e79e and released in `0.4.0`.

```
<plugin>
  <groupId>de.jjohannes</groupId>
  <artifactId>gradle-module-metadata-maven-plugin</artifactId>
  <configuration>
    <platformDependencies>
      <dependency>
        <groupId>org.junit</groupId>
        <artifactId>junit-bom</artifactId>
        <version>5.6.0</version>
      </dependency>
    </platformDependencies>
    <removedDependencies>
      <dependency>
        <groupId>ch.randelshofer</groupId>
        <artifactId>fastdoubleparser</artifactId>
      </dependency>
    </removedDependencies>
  </configuration>
</plugin>
```
